yarn.lock

```diff
@@ -13501,8 +13501,8 @@ semver-compare@^1.0.0:
   resolved "https://registry.npmjs.org/semver-compare/-/semver-compare-1.0.0.tgz"
 
 semver-regex@^3.1.2:
-  version "3.1.3"
-  resolved "https://registry.npmjs.org/semver-regex/-/semver-regex-3.1.3.tgz"
+  version "3.1.4"
+  resolved "https://registry.yarnpkg.com/semver-regex/-/semver-regex-3.1.4.tgz#13053c0d4aa11d070a2f2872b6b1e3ae1e1971b4"
```

Comment:
Looks good, but why it changes from npm registry to yarn package registry?

Reply:
I guess yarnpkg is the preferred registry now for Dependabot, although I can't find any material on the subject.